pkgs/tools/security/magnifier/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, llvmPackages_14
+, cmake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "magnifier";
+  version = "unstable-2022-08-26";
+
+  src = fetchFromGitHub {
+    owner = "trailofbits";
+    repo = "magnifier";
+    rev = "27c849db162e509aa413bdd971d7c55e32bc00a3";
+    sha256 = "sha256-yL71f3j9R52c1rFAsB9jPUoBASwdpyFeLSvSAAn9xms=";
+  };
+
+  patches = [ ./CMakeLists.txt.patch ];
+
+  nativeBuildInputs = [
+    llvmPackages_14.lld
```

Comment:
Are you sure this is needed? Can't `stdenv` handle this?

Reply:
you're right, it does handle it